cmd/devp2p/internal/ethtest/types.go

```diff
@@ -80,101 +77,74 @@ type Pong struct{}
 
 func (p Pong) Code() int { return 0x03 }
 
+type EthMessage interface {
+	eth.Packet
+}
+
 // Status is the network packet for the status message for eth/64 and later.
 type Status struct {
-	ProtocolVersion uint32
-	NetworkID       uint64
-	TD              *big.Int
-	Head            common.Hash
-	Genesis         common.Hash
-	ForkID          forkid.ID
+	*eth.StatusPacket
 }
 
 func (s Status) Code() int { return 16 }
 
 // NewBlockHashes is the network packet for the block announcements.
-type NewBlockHashes []struct {
-	Hash   common.Hash // Hash of one particular block being announced
-	Number uint64      // Number of one particular block being announced
+type NewBlockHashes struct {
+	*eth.NewBlockHashesPacket
```

Comment:
Might be better not to embed it here. If you just use

    type NewBlockHashes eth.NewBlockHashesPacket
    
you can still use the fields of the upstream definition, but don't need to wrap all the constructor call sites